### ISSUE TITLE
Is this library usable without installing coreos CRD ? (service monitors) 

### ISSUE THREAD

**Issue by kuberkaul** [DirectXMan12/k8s-prometheus-adapter#240]

**Comment by robinpercy** [comment]
@kuberkaul yes, this just needs a prometheus server to be setup. Doesn't matter it was via the Operator or not.

**Comment by kuberkaul** [comment]
@robinpercy great, how would you in that case scrape metrics of a deployment and send it to prometheus ? (if we dont have service monitors) ...

**Comment by robinpercy** [comment]
@kuberkaul If you're not using the operator, then you'll need to configure jobs directly. See https://blog.sebastian-daschner.com/entries/prometheus-kubernetes-discovery
